leveldb/table_test.go

```diff
@@ -40,7 +40,11 @@ func TestGetOverlaps(t *testing.T) {
 			copy(key, tmp)
 			return key
 		}
-		return []byte(makeInternalKey(nil, tmp, 0, typ))
+		ik, err := makeInternalKey(nil, tmp, 0, typ)
+		if err != nil {
+			panic(err) // Test helper - should not fail with valid parameters
+		}
```

Comment:
Here and below on lines 142-144, suggest `require.NoError(t, err, "making internal key")` or a similar test assertion instead